odoo/addons/base/models/ir_model.py

```diff
@@ -99,6 +100,9 @@ def _default_field_id(self):
 
     name = fields.Char(string='Model Description', translate=True, required=True)
     model = fields.Char(default='x_', required=True, index=True)
+    default_order = fields.Char(string='Order', default='id',
```

Comment:
Why *default* order?  It looks misleading in this context.  I would simply name it `order`.